com.github.jorchube.monitorets.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.github.jorchube.monitorets",
```

Comment:
It should be `io.github` not `com.github`

Reply:
I've changed it. Thanks for pointing it out.

Just out of curiosity: I was under the impression that using the `io.github.bla.bla` reverse dns should mean that I have setup a github page for the app.

Is it expected that it exists given the app id?

Thanks again for your help.

Reply:
no on github it's defacto yours `io.github.`USERNAME. It's find if it is not setup. It's just namespace ownership.